src/Credentials/ExternalAccountCredentials.php

```diff
@@ -34,6 +35,7 @@
 class ExternalAccountCredentials implements FetchAuthTokenInterface, UpdateMetadataInterface, GetQuotaProjectInterface
 {
     use UpdateMetadataTrait;
+    use MetricsTrait;
```

Comment:
Yes, any class not setting it's own $credType takes would mean the default `$credType = ''` value would be used.

This allows rest of the logic to avoid setting any metrics header for this particular case.

Reply:
I am surprised we don't have a credtype for this

Reply:
This thread is obsolete because we've removed the usage of `MetricsTrait` in `ExternalAccountCredentials`.